runtime/doc/syntax.txt

```diff
@@ -4789,10 +4791,10 @@ ModeMsg		'showmode' message (e.g., "-- INSERT --")
 							*hl-MoreMsg*
 MoreMsg		|more-prompt|
 							*hl-NonText*
-NonText		'~' and '@' at the end of the window, characters from
-		'showbreak' and other characters that do not really exist in
-		the text (e.g., ">" displayed when a double-wide character
-		doesn't fit at the end of the line).
+NonText		'@' at the end of the window, characters from 'showbreak' 
+		and other characters that do not really exist in the text 
+		(e.g., ">" displayed when a double-wide character doesn't 
+		fit at the end of the line).
```

Comment:
If a user finds NonText, wouldnt it be helpful to mention endofbuffer? They are related concepts...